dms_field/tests/test_dms_field.py

```diff
@@ -155,7 +155,7 @@ def test_creation_process_01(self):
         self.assertEqual(total, 1)
         # Create directory again (access groups are the same)
         template.create_dms_directory()
-        self.partner.refresh()
+        self.partner.invalidate_model()
```

Comment:
Why do we need to manually refresh? Isn't this got by depends?

Reply:
Sorry, which depends? I don't understand what you want to do here, this is necessary, it's only a test, it's like this in v15 and nothing has been commented on the migration of `dms_field` to v16.....until now.

Reply:
I know that it's the same as in v15, but you need to do an invalidation because data is not refreshed, and it's usually not refreshed because a computed field doesn't have proper dependencies, and thus, you need to manually trigger the recomputation. For what is this needed?

Reply:
There is no compute field (and therefore no depends) https://github.com/OCA/dms/blob/16.0/dms_field/models/dms_field_mixin.py#L12

Reply:
So there's no need on `invalidate_model`?

Reply:
Ok, deleted. According to https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0 you should change the `refresh()` method to `invalidate_model()` (a warning log also indicates this).

Reply:
Yeah, but seeing the change, I wonder why it's needed. Let's see if this affects in something in the tests. If so, then we would document at least why it's needed, or maybe fix the part why this is forced.

Reply:
It seems that the one2many that is fed into the create method is not accessible if you don't "refresh". That seems very weird. Is it happening the same on UI? As said, at least we can put a comment saying the reason for refreshing.

Reply:
In UX it is refreshed when the directory is created, https://github.com/OCA/dms/blob/16.0/dms_field/static/src/views/dms_list/dms_list_renderer.esm.js#L450.

Comments added in tests, is it correct now?